### PR TITLE
Jetpack Connect: install instructions are now rendered as html for i18n

### PR DESCRIPTION
**Pull request by rickybanister**
Previously we used .png files for each Jetpack Connect instruction step. This was faster to get it going for testing, but they were impossible to internationalize. 

**Before:**
screen shot 2016-06-06 at 3 35 01 pm

**After:**
screen shot 2016-06-06 at 3 35 18 pm
screen shot 2016-06-06 at 3 34 21 pm
screen shot 2016-06-06 at 3 34 35 pm

The rendering logic for the 'activate' step needs some help from @johnHackworth and @roccotripaldi

### CODE REVIEW

**Comment by roccotripaldi** [comment]
@rickybanister - logic fixed!

File: client/signup/jetpack-connect/style.scss [hunk 1]
```diff
@@ -170,6 +170,120 @@
 	.site-address-container {
 		margin-top: 8px;
 	}
+
+	.jetpack-connect__example-content {
```
Comment (mtias) [inline client/signup/jetpack-connect/style.scss]: It seems we should be able to drop most of the nesting?

Comment (rickybanister) [inline client/signup/jetpack-connect/style.scss]: I'll unnest it. I did a lot of refactoring as I went and some of the nesting stayed in.

File: client/signup/jetpack-connect/exampleComponents/jetpack-activate.jsx [hunk 2]
```diff
@@ -9,14 +10,31 @@ import Gridicon from 'components/gridicon';
 export default React.createClass( {
 	displayName: 'JetpackConnectExampleActivate',
 
+	renderActivate() {
+		if ( this.props.isInstall ) {
+			return (
+				<div className="jetpack-connect__example-content-wp-admin-plugin-card">
+					<div className="jetpack-connect__example-content-wp-admin-plugin-name" aria-hidden="true">{ this.translate( 'Jetpack by WordPress.com', { context: 'Jetpack Connect activate plugin instructions, plugin title' } ) }</div>
```
Comment (roccotripaldi) [inline client/signup/jetpack-connect/exampleComponents/jetpack-activate.jsx]: I'd move the translate calls to their own line
```
<div>
    { this.translate( ... ) }
</div>
```

File: client/signup/jetpack-connect/exampleComponents/jetpack-activate.jsx [hunk 3]
```diff
@@ -9,14 +10,31 @@ import Gridicon from 'components/gridicon';
...
+					<div className="jetpack-connect__example-content-wp-admin-plugin-activate-link" aria-hidden="true">{ this.translate( 'Activate', { context: 'Jetpack Connect activate plugin instructions, activate link' }  ) }</div>
```
Comment (roccotripaldi) [inline client/signup/jetpack-connect/exampleComponents/jetpack-activate.jsx]: see above

File: client/signup/jetpack-connect/exampleComponents/jetpack-activate.jsx [hunk 4]
```diff
@@ -9,14 +10,31 @@ import Gridicon from 'components/gridicon';
...
+				</div>
+			);
+		}
+		return (
+			<div className="jetpack-connect__example-content-wp-admin-activate-view">
+				<div className="jetpack-connect__example-content-wp-admin-activate-link" aria-hidden="true">{ this.translate( 'Activate Plugin', { context: 'Jetpack Connect activate plugin instructions, activate link' }  ) }</div>
```
Comment (roccotripaldi) [inline client/signup/jetpack-connect/exampleComponents/jetpack-activate.jsx]: see above

File: client/signup/jetpack-connect/exampleComponents/jetpack-connect.jsx [hunk 5]
```diff
@@ -26,9 +27,16 @@ export default React.createClass( {
...
 					</div>
-
 				</div>
-				<img src="/calypso/images/jetpack/jetpack-connect-connect.png" />
+				<div className="jetpack-connect__example-content jetpack-connect__example-connect-jetpack">
+					<div className="jetpack-connect__example-content-wp-admin-masterbar"></div>
+					<div className="jetpack-connect__example-content-wp-admin-sidebar"></div>
+					<div className="jetpack-connect__example-content-wp-admin-main">
+						<div className="jetpack-connect__example-content-wp-admin-connect-banner">
+							<div className="jetpack-connect__example-content-wp-admin-connect-button" aria-hidden="true">{ this.translate( 'Connect to WordPress.com', { context: 'Jetpack Connect post-plugin-activation step, Connect to WordPress.com button' }  ) }</div>
```
Comment (roccotripaldi) [inline client/signup/jetpack-connect/exampleComponents/jetpack-connect.jsx]: see above

File: client/signup/jetpack-connect/exampleComponents/jetpack-install.jsx [hunk 6]
```diff
@@ -26,9 +27,14 @@ export default React.createClass( {
...
 							placeholder={ this.props.url } />
 					</div>
-
 				</div>
-				<img src="/calypso/images/jetpack/jetpack-connect-install.png" />
+				<div className="jetpack-connect__example-content jetpack-connect__example-install-jetpack">
+					<div className="jetpack-connect__example-install-plugin-header"></div>
+					<div className="jetpack-connect__example-install-plugin-body"></div>
+					<div className="jetpack-connect__example-install-plugin-footer">
+						<div className="jetpack-connect__example-install-plugin-footer-button" aria-hidden="true">{ this.translate( 'Install Now', { context: 'Jetpack Connect install plugin instructions, install button' }  ) }</div>
```
Comment (roccotripaldi) [inline client/signup/jetpack-connect/exampleComponents/jetpack-install.jsx]: see above

**Comment by roccotripaldi** [comment]
In my testing I was only able to get the 'Activate Plugin' view.
How do I see the other view?

**Comment by roccotripaldi** [comment]
I figured it out! I pushed a small change. I think it is ready to go. 🚢

**Pull request merged**